db/migrate/20191128003907_add_clearance_to_users.rb

```diff
@@ -0,0 +1,26 @@
+class AddClearanceToUsers < ActiveRecord::Migration[6.0]
+  def self.up
+    change_table :users do |t|
+      t.string :confirmation_token, limit: 128
+      t.string :remember_token, limit: 128
+    end
+
+    add_index :users, :remember_token
+
+    users = select_all("SELECT id FROM users WHERE remember_token IS NULL")
+
+    users.each do |user|
+      update <<-SQL
+        UPDATE users
+        SET remember_token = '#{Clearance::Token.new}'
```

Comment:
Nifty! Was this in the README or did you source dive to find this class name?

Reply:
This was in the README, created when I ran 'rails generate clearance:install'